### DISCUSSION TITLE
ARK multiplayer mode not available

### DISCUSSION THREAD

**Discussion by ErikBraun251** [legendary-gl/legendary#464] [Q&A]
Hey there!
Thank you for your great software. I've finally managed to get "ARK Survival Evolved" (from Epic Store) running on my M2 MacOS. However it seems that I'm still missing one final piece, because the multiplayer option (to connect to servers) is not available. I've tried starting ARK with `legendary launch ark` (alias set) and `legendary launch ark --crossover`. But both only give me the offline/singleplayer option in the start menu. 
Maybe someone can help me here...?

**Comment by nutterthanos** [comment]
Um does trying to launch multiplayer via EGL work tho?

**Reply by ErikBraun251** [reply]
When I start Ark via EGL (Win) on my Windows PC, multiplayer works. But EGL on Mac will only download the (broken) Mac Version of ARK, which does not start at all - which was the reason in the first place, why I was looking for alternatives like legendary... :)

**Reply by nutterthanos** [reply]
> When I start Ark via EGL (Win) on my Windows PC, multiplayer works. But EGL on Mac will only download the (broken) Mac Version of ARK, which does not start at all - which was the reason in the first place, why I was looking for alternatives like legendary... :)

In this case it might possibly be a bug with the game more than anything

**Reply by TheAlpha313** [reply]
i can't guarantee but i believe it's neither the game or the launcher. It's something with the crossover i think, every time i run GTA through legendary it launches in offline mode so strangely i have to enable vpn, i use turbo vpn before executing the command "legendary launch gtav" and all online features start working, and then even if i disable the vpn and completely quit the vpn the game remains in online state as long as it's running so can give it a try and i hope it'll be the same case for you cuz at least better than broken game yk, and please let me know cuz i wonna try ARK to
